datadog_checks_dev/datadog_checks/dev/tooling/commands/console.py

```diff
@@ -6,6 +6,7 @@
 
 import click
 
+from ...ci import running_on_gh_actions  # noqa: F401
```

Comment:
```suggestion
from ...ci import running_on_gh_actions
```

Reply:
Just curious, what does `noqa: F401` mean?

Reply:
It ignores style check for unused imports